docs/examples.rst

```diff
@@ -176,6 +176,7 @@ You can also update an issue's fields with keyword arguments::
 
     issue.update(summary='new summary', description='A new summary was added')
     issue.update(assignee={'name': 'new_user'})    # reassigning in update requires issue edit permission
+    issue.update(reporter={'name'; 'new_user'})    # changing the reporter of an issue
```

Comment:
Typo... there should be a colon and not a semi-colon
```python
issue.update(reporter={'name': 'new_user'})    # changing the reporter of an issue
```

Reply:
Thanks for the hint. The typo is now fixed.